util/syspolicy/policy_keys.go

```diff
@@ -24,6 +24,7 @@ const (
 	UpdateMenuVisibility      Key = "UpdateMenu"
 	RunExitNodeVisibility     Key = "RunExitNode"
 	PreferencesMenuVisibility Key = "PreferencesMenu"
+	AllowLANAccessVisibility  Key = "AllowLANAccess"
```

Comment:
I think this should be a always/never/user-decides, not a show/hide?

Reply:
good point. @agottardo thoughts? thanks!

Reply:
@clairew agreed! This should be always/never/u-d.